src/Cake.Unity/SeekersOfEditors/SeekerOfEditors.cs

```diff
@@ -34,11 +34,9 @@ protected SeekerOfEditors(ICakeEnvironment environment, IGlobber globber, ICakeL
 
         public IReadOnlyCollection<UnityEditorDescriptor> Seek()
         {
-            var searchPattern = SearchPattern;
-
             log.Debug("Searching for available Unity Editors...");
-            log.Debug("Search pattern: {0}", searchPattern);
-            var candidates = globber.GetFiles(searchPattern).ToList();
+            log.Debug("Search patterns: {0}", SearchPatterns);
```

Comment:
I believe that you will not see search patterns in log here. `Array.ToString` does not behave that way.